lib/atlas/graph_persistor.rb

```diff
@@ -1,17 +1,20 @@
 module Atlas
   class GraphPersistor
-    def initialize(dataset, path)
-      @dataset = dataset
-      @path    = path
+    def initialize(dataset, path, export_modifier: nil)
+      @dataset        = dataset
+      @path           = path
+      @export_modifier = export_modifier
     end
 
-    def self.call(dataset, path)
-      new(dataset, path).persist!
+    def self.call(dataset, path, export_modifier: nil)
+      new(dataset, path, export_modifier: export_modifier).persist!
     end
 
     def persist!
+      data = EssentialExporter.dump(refinery_graph)
+      @export_modifier.call(data) if @export_modifier
```

Comment:
Well, not at all _at the moment_, but since it is an optional attribute, the code should be able to handle the blank case.